platform/src/apis/Platform.Api.Insight/Census/CensusAverageHistoryResponse.cs

```diff
@@ -0,0 +1,16 @@
+namespace Platform.Api.Insight.Census;
+
+public record CensusAverageHistoryResponse
+{
+    public string? Urn { get; set; }
+    public int? Year { get; set; }
+    public decimal? TotalPupils { get; set; }
+    public decimal? Workforce { get; set; }
+    public decimal? WorkforceHeadcount { get; set; }
+    public decimal? Teachers { get; set; }
+    public decimal? SeniorLeadership { get; set; }
+    public decimal? TeachingAssistant { get; set; }
+    public decimal? NonClassroomSupportStaff { get; set; }
+    public decimal? AuxiliaryStaff { get; set; }
+    public decimal? PercentTeacherWithQualifiedStatus { get; set; }
+}
```

Comment:
We will need 'term' in here too, I think, but I'm not sure if this is actually just doing the same as `CensusHistoryResponse`